assets/js/webpack-entry.js

```diff
@@ -48,10 +48,6 @@ const comparePublishDates = (a, b) => {
  */
 new Vue({
```

Comment:
Once this is done, I think we can go ahead and remove all the Vue-specific code in this file including this main Vue instance. We'll also need to remove the main Vue import `require('vue/dist/vue.min')` above.